tools/setup_helpers/env.py

```diff
@@ -59,3 +59,6 @@ def hotpatch_build_env_vars():
 DEBUG = check_env_flag('DEBUG')
 REL_WITH_DEB_INFO = check_env_flag('REL_WITH_DEB_INFO')
 NAMEDTENSOR_ENABLED = check_env_flag('USE_NAMEDTENSOR') or check_negative_env_flag('NO_NAMEDTENSOR')
+USE_CBLAS_IN_MKLDNN = 'OFF'
+if check_env_flag('USE_MKLDNN'):
+    USE_CBLAS_IN_MKLDNN = check_env_flag('USE_CBLAS_IN_MKLDNN')
```

Comment:
You probably do not need these three lines because environment variables starting with `USE_` will be automatically passed to CMake (there has been some changes in the build system since last review).

Reply:
Thanks for your reminder. I have removed these three lines and add the "MKLDNN_CBLAS" in "use_env_vars" so that it will automatically check the environment. Please help review. Thanks.